src/ui/components/Rating/styles.scss

```diff
@@ -69,6 +69,7 @@
   .Rating-star {
     background-image: url('./img/open-star-dim-yellow.svg');
     cursor: pointer;
+    outline: none;
```

Comment:
As @SeanPrashad suggested, @HosseinMarzban  we need to revert it as PR #6762 is looking to solve the issue #6755.